build/build-sdk-images/node/Dockerfile

```diff
@@ -16,8 +16,23 @@ FROM $BASE_IMAGE
 
 ## --allow-releaseinfo-change is a workaround for the new debian release
 ## Can be removed when upgrading to bullseye
-RUN apt-get --allow-releaseinfo-change update && curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
-    apt-get install -y nodejs
+RUN apt-get update && apt-get install -y curl && apt-get install -y software-properties-common && \
```

Comment:
I think either the comment could be removed or we are missing the --allow-releaseinfo-change flag?

Reply:
Done.